vote_text_drafts/clarify_RPIP-4.md

```diff
@@ -0,0 +1,42 @@
+> Vote options:
+> - All nodes shall be eligible voters
+> - Nodes w/RPL ≥10% borrowed ETH shall be eligible voters
+> - Abstain
+
+# Clarify Voting Criteria (RPIP-4) [eligible voters: all nodes]
+
+There are two concurrent votes to clarify voting criteria:
+- `Clarify Voting Criteria (RPIP-4) [eligible voters: all nodes]`
+- `Clarify Voting Criteria (RPIP-4) [eligible voters: nodes w/RPL ≥10% borrowed ETH]`
+
+These votes are identical except that different strategies were used to determine voter eligibility.
+
+This vote will determine eligible voters for future pDAO governance votes.
+
+### Context around the situation
+
+RPIP-4 defines who gets to vote: `To be eligible for vote participation, the RPL must be effectively staked in the Rocket Pool protocol as reported by getNodeEffectiveRPLStake().`
+
+This is currently interpreted by people multiple ways, including at least:
+- It was meant to be effective stake (10-150%); the function was not there as a definition, but as an implementation that was bugged. Fixing the bugged function (and vote weight) was proper.
+- Effective stake is defined in the RPIP using the function at the time. Fixing the bugged function (and vote weight) was inappropriate without pDAO input.
+- It's ambiguous.
+
+The voting body _was_ changed with Atlas release - whether that was ok depends on where you land above.
+Since this is about voting, we can't simply vote to figure out what we want -- the set of people that should vote on that is in question (ie, this is a bootstrap problem). This is why the approach is to vote using both possible voting bodies to get this clarified.
```

Comment:
I might reword this. Given that we _are_ voting about it to figure out what we want (though not simply.)